### PR TITLE
Update dotnet SDK to 2.1.300-rtm-008866

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.300-preview2-008324"
+    "version": "2.1.300-rc1-008673"
```
Comment (jaredpar) [inline global.json]: When changin the SDK we also need to update the Contributing To Windows document + email the team notifying them that an update is needed.

**Comment by jcouv** [comment]
Thanks for the email with install link: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.300-rc1-008673/dotnet-sdk-2.1.300-rc1-008673-win-x64.exe

If I recall correctly, there is an error message that includes this URL (to make the install link discoverable to contributors).

File: global.json (hunk 1 above)
Comment (tmat) [reply global.json]: Updated

**Comment by jcouv** [comment]
Out of curiosity, is there a feature that we want to use from the new SDK, which is prompting this change?

Also, it looks like there are test failures that might be related to the new SDK...

**Comment by nguerrera** [comment]
I would add that in addition to getting source link feature, it's a big help to get some dogfooding on the RC from Roslyn.

**Comment by tmat** [comment]
test ubuntu_16_debug_prtest please

**Comment by tmat** [comment]
@jcouv The CI is happy now :)

**Comment by alrz** [comment]
I can build with command line but not VS.

image


@tmat How can I make VS happy?

**Comment by jcouv** [comment]
@alrz For me, installing the linked SDK and restarting VS fixed those errors.

**Comment by alrz** [comment]
>For me, installing the linked SDK and restarting VS fixed those errors.

happy to see that could work. Next time I'll try that. Now I had to start over, (cleaning nuget etc).

**Comment by jcouv** [comment]
@alrz Now that you mention it, it possible that I also did a `git clean -xdf`. I'm not sure.

**Pull request merged**